### PR TITLE
add a11y labels to calendar header

### PR DESCRIPTION
**Pull request by amolgupta**
Adding acessibility lables to weekday header view. Will fix the date views in a seprate PR

+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: Backpack/src/main/java/net/skyscanner/backpack/calendar/view/WeekdayHeaderView.kt [hunk 1]
```diff
@@ -47,6 +47,8 @@ internal class WeekdayHeaderView @JvmOverloads constructor(
 
   internal fun initializeWithLocale(locale: Locale) {
     val formatter = DateTimeFormatter.ofPattern("ccc", locale)
+    val descriptionFormatter = DateTimeFormatter.ofPattern("cccc", locale)
```
Comment (tiagoengel) [inline Backpack/src/main/java/net/skyscanner/backpack/calendar/view/WeekdayHeaderView.kt:50]: Why `cccc`?

File: Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt [hunk 2]
```diff
@@ -186,6 +186,7 @@ internal class MonthView @JvmOverloads constructor(
     controller?.let {
       drawMonthTitle(canvas)
       drawDaysInMonth(it, canvas)
+      contentDescription = monthHeaderString
```
Comment (tiagoengel) [inline Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt:189]: Why is this necessary?

File: Backpack/src/main/java/net/skyscanner/backpack/calendar/view/WeekdayHeaderView.kt (hunk 1 above)
Comment (amolgupta) [reply Backpack/src/main/java/net/skyscanner/backpack/calendar/view/WeekdayHeaderView.kt:50]: For screenreader it would speak the complete word. eg `Monday` and not `Mon`

File: Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt (hunk 2 above)
Comment (amolgupta) [reply Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt:189]: Unllike `textview` ,  the `paint` object does not set the content description automatically. Adding this will speak the month name correctly.
